.circleci/config.yml

```diff
@@ -0,0 +1,89 @@
+version: 2.1
+references:
+  persist_to_workspace: &PERSIST_TO_WORKSPACE
+    persist_to_workspace:
+      root: ~/project
+      paths: .
+  attach_workspace: &ATTACH_WORKSPACE
+    attach_workspace:
+      at: ~/project
+commands:
+  run_tests:
+    steps:
+      - run: npm test
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.15
+    steps:
+      - checkout
+      # TODO: remove npm auth after PR #63 resolution
```

Comment:
would this actually be removed as it needs permission to publish to npm?

Reply:
We'd have it as part of of the publish step only